src/main/java/pro/trevor/tankgame/rule/impl/version3/Ruleset.java

```diff
@@ -85,7 +85,10 @@ public void registerPlayerRules(RulesetDescription ruleset) {
         playerRules.put(Tank.class, PlayerRules.GetShareGoldWithTaxRule(1));
         playerRules.put(Tank.class, PlayerRules.SPEND_ACTION_TO_MOVE);
         playerRules.put(Tank.class, PlayerRules.SpendActionToShootWithDeathHandle(LineOfSight::hasLineOfSightV3,
-            (s, t, d) -> t.setGold(t.getGold() + d.getGold() + d.getBounty())
+            (s, t, d) -> {
+                t.setGold(t.getGold() + d.getGold() + d.getBounty());
+                s.getCouncil().setCoffer(s.getCouncil().getCoffer() + 1);
```

Comment:
I don't think the council gets anything upon looting in v3. Not sure why this is being changed.

Reply:
DONE